src/components/BrowserEventList/EventDescription/InputChangeDescription.tsx

```diff
@@ -1,34 +1,28 @@
 import { css } from '@emotion/react'
-import { EyeNoneIcon, EyeOpenIcon } from '@radix-ui/react-icons'
+import { EyeIcon, EyeOffIcon } from 'lucide-react'
 import { ReactNode, useState } from 'react'
 
 import { InputChangeEvent } from '@/schemas/recording'
 import { HighlightSelector } from 'extension/src/messaging/types'
 
 import { Selector } from './Selector'
 
-const valueStyles = css`
-  display: inline-flex;
-  align-items: baseline;
-  gap: var(--space-1);
-`
-
 const buttonStyles = css`
   display: inline-flex;
   align-items: center;
-  justify-content: center;
   border: none;
   background: transparent;
-  padding: 0;
+  padding: 0 0.3em;
   cursor: pointer;
+  color: var(--studio-foreground);
```

Comment:
I noticed that the icon wasn't visible in dark mode, so I snuck a little fix in.